packit_service/worker/comment_action_handler.py

```diff
@@ -53,53 +49,16 @@ def add_to_comment_action_mapping(kls: Type["CommentActionHandler"]):
     return kls
 
 
-class CommentActionHandler:
+class CommentActionHandler(Handler):
     name: CommentAction
 
     def __init__(
         self,
         config: ServiceConfig,
         event: Union[PullRequestCommentEvent, IssueCommentEvent],
     ):
-        self.config: ServiceConfig = config
+        super().__init__(config)
         self.event: Union[PullRequestCommentEvent, IssueCommentEvent] = event
-        self.api: Optional[PackitAPI] = None
-        self.local_project: Optional[PackitAPI] = None
 
     def run(self) -> HandlerResults:
         raise NotImplementedError("This should have been implemented.")
-
-    def run_n_clean(self) -> HandlerResults:
-        try:
-            return self.run()
-        finally:
-            self.clean()
-
-    def _clean_workplace(self):
```

Comment:
why did you *remove* the two clean methods?

Reply:
There's now only one in common ancestor class [Handler](https://github.com/packit-service/packit-service/pull/200/files#diff-326013bd647064da42d86cad54398b7eR136)